### PR TITLE
Eg 99 fix console warnings

### LINKED ISSUE #99: Fix Console Warnings
## Expected Behavior
There should be no extreme console warnings coming from the browser.

## Actual/Current Behavior
The console is throwing 3 warnings to the browser:

1. `Warning: Received ``true`` for a non-boolean attribute ``enabled`` `.
2. `Uncaught Error: The error you provided does not container a stack trace`.
3. `Uncaught (in promise) DOMException: The play() request was interrupted by a new load request`.

## Screenshots
Screenshot 2021-08-09 103414

### PR DESCRIPTION
**Pull request by BlackCubes**
## Changes
1. Removed the `enabled` attribute in `button` element in `ThemeSongs.js` which was throwing a warning to the browser console.
2. Refactored the `ThemeSongs` component with its functionalities to render in `Main.js` and not in `Search.js`, and this would prevent the warning to the console.
3. Implemented the `audioPlayAsync` utility function in `ThemeSongs.js` to prevent any potential future warnings about the browser not accepting the `play()` method.
4. Removed `isDisplay` states in `Versus.js` since it was not being used, and to remove the warning on the console about unused variables.

## Purpose
There were warnings being thrown to the console in the browser.

## Approach
There were a few warnings being thrown to the console where one of them stating that the `enabled` attribute in the `button` element in `ThemeSongs.js` was receiving a `boolean` when it wants a `string` value. To fix this is simply remove it since it is not being used.

Another warning was that the `play()` request in `ThemeSongs.js` was interrupted by a new load request, and this was due to the fact that the `ThemeSongs` component was residing in `Search.js`. This would cause multiple re-renders as the user types which would then lead to the warning in the console. To fix this is to put the `ThemeSongs` component in `Main.js` along with any functionalities related to it (such as states) so that it would not re-render multiple times, and thus remove the warning.

As a precaution, the `audioPlayAsync` utility function was implemented in `ThemeSongs.js` in order to prevent any future warnings/errors to the browser such as the possibility that the user disabled the audio.

Lasly, `isDisplay` was not implemented in `Versus.js` which was thus throwing a warning. A simple fix is to remove it along with any unused functionalities associated with it.

Closes #99

### CODE REVIEW

File: pokemon-app/src/components/ThemeSongs.js [hunk 1]
```diff
@@ -18,18 +30,25 @@ const ThemeSongs = ({ src, isPlaying, setIsPlaying }) => {
 
   useEffect(() => {
     if (isPlaying) {
-      audioPlayer.current.volume = 0.08;
-      audioPlayer.current.play();
+      if (audioPlayer.current) {
+        audioPlayer.current.volume = 0.08;
+        audioPlay(audioPlayer.current);
+      } else {
+        console.error("audioPlayer.current not working...");
```
Comment (Lap343) [inline pokemon-app/src/components/ThemeSongs.js:37]: nice putting in the console.error()

**Review by Lap343 — APPROVED** [review]
Good job on the PR code. No more annoying error

**Review by connielion — APPROVED** [review]
Very cool stuff going on. Keep up the great work!

**Review by Wolftownn — APPROVED** [review]
Great job on this Elias! Those error codes didn't want to go away.

**Pull request merged**